content/applications/productivity/mail_plugins/outlook.rst

```diff
@@ -75,6 +75,21 @@ Connect your database
       Only a limited amount of *Company Insights* (*Lead Enrichment*) requests are available as a
       trial. This feature requires :ref:`prepaid credits <mail_plugins/pricing>`.
 
+   .. tip::
+      If, after a short while, the panel is still empty, it is possible that your browser cookie
+      settings prevented it from loading.
+      Note that these settings also change if you are in "Incognito" mode on your
+      browser.
+
+      To fix this issue, configure your browser to always allow cookies on Odoo's plugin page.
+
+      For Google Chrome, you can do so by following the guide at:
+      `https://support.google.com/chrome/answer/95647 <https://support.google.com/chrome/answer/95647#:~:text=Allow%20or%20block%20cookies%20for%20a%20specific%20site>`_
```

Comment:
Nitpicking for your future PRs: [you can break the line anywhere inside a link label](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character).
```suggestion
      `https://support.google.com/chrome/answer/95647
<https://support.google.com/chrome/answer/95647#:~:text=Allow%20or%20block%20cookies%20for%20a%20specific%20site>`_
```